modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -642,16 +719,29 @@ The `SET PASSWORDS` privilege allows the user to run the `ALTER USER` administra
 ALTER USER jake SET PASSWORD 'abcd5678' CHANGE NOT REQUIRED
 ----
 
-A user that is granted the `SET AUTH` privilege is allowed to run the `ALTER USER` administration command with one or both of the `SET AUTH` and `REMOVE AUTH` parts:
+=== Grant privilege to modify users' auth providers
+
+You can grant the privilege to modify users' auth providers using the `SET AUTH` privilege. +
```

Comment:
```suggestion
You can grant the privilege to modify users' auth information using the `SET AUTH` privilege. +
```
since you can change both the provider and the relevant clauses for that provider (id or password)